dnsserv_test.go

```diff
@@ -1,40 +0,0 @@
-package main
-
-import (
-	"bytes"
-	. "gopkg.in/check.v1"
-	"net"
-	"testing"
-)
-
-func Test(t *testing.T) { TestingT(t) }
-
-type DNSNameserverSuite struct{}
-
-var _ = Suite(&DNSNameserverSuite{})
-
-func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
```

Comment:
There are different styles of implementation. Also, we do know what the bytes will look like, right? I'm confused why they would be different than what is specified in the DNS spec. 

I don't mind quick spikes to test connecting, but not covering actual implementation code with tests as we go along isn't such a good idea :/